tsconfig.base.json

```diff
@@ -17,6 +17,9 @@
 		"baseUrl": ".",
 		"paths": {
 			"@recipes-api/cwmetrics": ["lib/cwmetrics/src/index.ts"],
+			"@recipes-api/lambda/recipes-reindex": [
+				"lambda/recipes-reindex/src/index.ts"
+			],
```

Comment:
Not sure that i understand this - is the package intended as a library or an executable?

Reply:
Mistakenly created as a lib first, and missed this on cleanup, thanks for spotting. dc4ceac0892184524353edc87c216f0b3545a4c4